File: Modul 4/Bahasa Python/PRAK405-2310817120001-DinaIzzatiElfadheya.py
print("Input")
banyak_baris, kelipatan = map(int, input().split())
total = 0
#barisnya error
print("\nOutput")

for i in range(1, banyak_baris + 1):
    baris_total = 0
    for bilangan in range(i, 0, -1):
        nilai_kelipatan = bilangan * kelipatan
        print(f"({bilangan} * {kelipatan})", end="")
        if bilangan > 1:
            print(" + ", end="")
        baris_total += nilai_kelipatan
    total += baris_total
    print(f" = {baris_total}")

print(total)

File: Modul 4/Bahasa C/PRAK405-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>
int main() {
    int banyak_baris, kelipatan;
    int total = 0;

    printf("Input\n");
    scanf("%d %d", &banyak_baris, &kelipatan);
    printf("\nOutput\n");
    
    for (int i = 1; i <= banyak_baris; i++) {
        int baris_total = 0;
        for (int bilangan = i; bilangan >= 1; bilangan--) {
            int nilai_kelipatan = bilangan * kelipatan;
            printf("(%d * %d)", bilangan, kelipatan);
            if (bilangan > 1) {
                printf(" + ");
            }
            baris_total += nilai_kelipatan;
        }
        total += baris_total;
        printf(" = %d\n", baris_total);
    }
    printf("%d \n", total);
    return 0;
}
